src/constructs/loadbalancing/elb.ts

```diff
@@ -44,8 +40,11 @@ interface GuClassicLoadBalancerProps extends Omit<LoadBalancerProps, "healthChec
  * ```
  * If you are running an application which only accepts traffic over HTTPs, consider using [[`GuHttpsClassicLoadBalancer`]]
  * to reduce the amount of boilerplate needed when configuring your load balancer.
+ *
+ * This resource is stateful.
+ * @see https://github.com/guardian/cdk/blob/main/docs/stateful-resources.md
  */
-export class GuClassicLoadBalancer extends GuStatefulMigratableConstruct(GuAppAwareConstruct(LoadBalancer)) {
+export class GuClassicLoadBalancer extends GuAppAwareConstruct(LoadBalancer) {
```

Comment:
Also one for another PR... I wonder if we should remove this construct entirely?